flatcontent/tests/test_flatcontent.py

```diff
@@ -55,6 +55,19 @@ def test_get_not_found(self):
         # Test the flatcontent doesn't exist
         eq_('', FlatContent.get('not-found'))
 
+    def test_get_with_context(self):
+        flatcontent = FlatContentFactory.create(
+            slug='test-tpl', site=None, content='Hi {{ name }}')
+
+        assert FlatContent.get('test-tpl') == 'Hi '
+
+        # Clear the cache
+        flatcontent.save()
```

Comment:
@grokcode I suppose ideally, but the only way I can think of to make that happen is to add the context to the cache key. I envisioned that this would be used with the same context every time